cfme/tests/cli/test_appliance_console.py

```diff
@@ -27,6 +27,14 @@
 ]
 
 
+@pytest.fixture(scope="function")
+def temp_appliance_time_crud(temp_appliance_preconfig_funcscope):
```

Comment:
**Required:** This function name is not very descriptive I think.   Maybe instead of temp_appliance_time_crud call it preconfig_appliance_time_crud or preconfig_appliance_time.   I'm also not sure what this has to do with crud in and of itself.   Not saying my names are the best, but the function name should very clearly state what the function is doing on some level.

Reply:
Oh, I'm beginning to understand what this function is all about.   You are returning a temp appliance with its time pre-configured.   Hard to find a good name for that in my head.   crud throws me because it makes me think you are going through all those operations.   Perhaps temp_appliance_with_base_time is better?